README.md

```diff
@@ -29,6 +29,41 @@ to the wider research and game developer communities.
 * Visualizing network outputs within the environment
 * Simplified set-up with Docker (Experimental)
 
+## Quick Start Guide
```

Comment:
How much of a duplicate is this with `Getting started with balance ball`

Reply:
The purpose of this addition is to allow people to get a taste of how ML-Agent works within 15 minutes (excluding the download time), with just 1 page of reading (instead of going through several pages)
A lot of other projects have this which is really useful. 
eg. 
1. https://github.com/deepmind/pysc2
2. https://github.com/openai/gym

Reply:
Should we maybe eliminate the balance ball doc then?

Reply:
The balance ball doc contains four parts
1. a link to Installation
2. a section talks about what academy, brain and agent are. 
3. a section talks about training, including tensorflowboard, parameter explanation
4. a section talks about doing inference. 

This would be too lengthy for a quick start, a quick start is targeted towards a different audience group, someone who just want to spend 15 minutes playing around with the project to see what it can do, without needing to know all the details of the project. 

But we still want to keep the balance ball page, since that is a really good material for people who has went through the quickstart to understand more details about the project. The quick start will tell people what they can do to start, and there will be links to balance ball so that people can know more if they want to.